test/integration/read-write-concern/write_concern.test.ts

```diff
@@ -94,6 +99,7 @@ describe('Write Concern', function () {
 
         afterEach(async function () {
           await db.dropDatabase();
+          await sleep(1000);
           await client.close();
```

Comment:
By no longer waiting for the server's response, if we plan on relying on an unacknowledged write immediately after its run, we need to set a timeout.

Reply:
Should this be included in the release notes / API docs?

Reply:
Putting sleeps in tests with arbitrary values is generally going to be flaky at times so I would avoid this. In this specific case I'm not sure why it's even necessary - why do we need to sleep before closing the client?